include/TI/soc/tivx_config_j784s4.h

```diff
@@ -505,6 +505,10 @@ extern "C" {
                                             TIVX_META_FORMAT_MAX_OBJECTS+ \
                                             TIVX_PARAMETER_MAX_OBJECTS)
 
+/*! \brief Max number of associated references for a graph parameter
+ * \ingroup group_vx_graph_cfg
+*/
+#define TIVX_GRAPH_MAX_PARAM_REFS          (8u)
```

Comment:
@lcweaver , I think there are implications when adding a new max value, in terms of adding it to the resource tracking and reporting, as well as boundary testing.  Can you point out what needs to be done for this?